tests/test_fix_dict_six.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import
 
-from utils import check_on_input
+from libmodernize.tests.utils import check_on_input
```

Comment:
The tests aren't actually part of libmodernize, so this is causing a failure.

Reply:
sorry, forgot to revert that before pushing!